README.md

```diff
@@ -65,6 +65,14 @@ Test the application on url http://localhost:8080/
 You can also use sbt instead of gradle. It offers a better integration with intelliJ and allows debug.
 
 
+## Using local repositories
+If you do not want to clone the repositories with git you can also specify the local pathes of the repositories :
+```
+local {
+  repositories = ["/path/to/repo1", "/path/to/repo2"]
```

Comment:
Small question, does it support directory scanning ? like retrieving all git projects in `/path/to` (my scala understanding is not top-notch)

Reply:
Yes it does, the findRepositories is recursive and if a directory contains only other directories it will inspect the sub directories